api/schemas/Room.yaml

```diff
@@ -4,7 +4,16 @@ properties:
   id:
     type: integer
     format: int64
+    readOnly: true
     example: 10
   description:
     type: string
-    example: Large Room with Low Ceilings
\ No newline at end of file
+    example: Large Room with Low Ceilings
+  features:
+    type: array
+    items: 
+      anyOf:
+        - $ref: "./BooleanFeature.yaml"
+        - $ref: "./NumericalFeature.yaml"
```

Comment:
We should define a schema for `Feature`, `BooleanFeature`, and `NumericalFeature`